extension/sqlsmith/include/statement_generator.hpp

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include "duckdb.hpp"
+#include "duckdb/parser/parsed_data/detach_info.hpp" // this should be included somewhere else
```

Comment:
does the comment still need to be here?
I think it's find to have this include statement no?

Reply:
I didn't find a better place for this include statement, so I'll just remove my comment